span.go

```diff
@@ -134,7 +163,21 @@ func (s *span) Log(ld opentracing.LogData) {
 		if ld.Timestamp.IsZero() {
 			ld.Timestamp = s.tracer.timeNow()
 		}
-		s.logs = append(s.logs, ld)
+		// TODO replace lr with ld.ToLogRecord() once nil payload is handled there
+		lr := opentracing.LogRecord{
+			Timestamp: ld.Timestamp,
+		}
+		if ld.Payload == nil {
+			lr.Fields = []log.Field{
+				log.String("event", ld.Event),
+			}
+		} else {
+			lr.Fields = []log.Field{
+				log.String("event", ld.Event),
+				log.Object("payload", ld.Payload),
+			}
+		}
+		s.logs = append(s.logs, lr)
```

Comment:
Per the TODO, this code isn't event meant to be here, once it moves to opentracing proper.